src/collections/_documentation/performance/alerts-notifications/notifications.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Other Notifications
+sidebar_order: 2
+---
+
+Aside from Alerts, Sentry has other types of notifications, such as Workflow, Deploy, and Quota notifications. All these notifications are sent via email.
+
+## Workflow
+
+Workflow relates to activity involving user actions and state changes on issues and alerts. This includes things like resolution, assignment, comments, and regressions. By default, Sentry subscribes you to issues it deems you're participating in, based on signals. 
```

Comment:
Suggest we make signals more clear. 

...By default, Sentry subscribes you to issues based signals, such as:

- Subscription to the issue or...
- Involvement in the commit related to...
- Comment or bookmarking the issue
- Mention of you or your team in the issue or assigned to the issue